### ISSUE TITLE
Pretrained FFHQ VQGAN and CelebA-HQ VQGAN

### ISSUE THREAD

**Issue by gulcinbaykal** [CompVis/taming-transformers#138]
Hi,

Thank you for your great work and repo. I need to use pretrained VQGAN models for FFHQ and CelebA-HQ datasets, separately. You already shared faceshq-vqgan, but i specifically need the Discriminators of VQGANs trained only with FFHQ or CelebA-HQ, not FacesHQ. As far as I understand, it is not possible to get the Discriminators from ffhq-transformer and celebahq-transformer configs since the loss configs are DummyLoss in both of them.

Do you plan to share the VQGANs trained with FFHQ and CelebA-HQ? It would be great if you share them too. Thank you in advance.

Bests

**Comment by tenmiles0** [comment]
Hello, have you obtained the pre-training parameters of FFHQ? I also hope to study based on it.

**Comment by gulcinbaykal** [comment]
> Hello, have you obtained the pre-training parameters of FFHQ? I also hope to study based on it.

Unfortunately no
